diff --git a/filter/source/config/cache/typedetection.cxx b/filter/source/config/cache/typedetection.cxx
index 065fe483a3dd..7baebf80c049 100644
--- a/filter/source/config/cache/typedetection.cxx
+++ b/filter/source/config/cache/typedetection.cxx
@@ -952,20 +952,24 @@ OUString TypeDetection::impl_detectTypeFlatAndDeep(      utl::MediaDescriptor& r
                             = aURL.getName(INetURLObject::LAST_SEGMENT, true,
                                            INetURLObject::DecodeMechanism::WithCharset);

+#if 0
                         // Ask the user whether they wants to try to repair
                         RequestPackageReparation aRequest(aDocumentTitle);
                         xInteraction->handle(aRequest.GetRequest());

                         if (aRequest.isApproved())
                         {
+#endif
                             // lok: we want to overwrite file in jail, so don't use template flag
                             const bool bIsLOK = comphelper::LibreOfficeKit::isActive();
                             rDescriptor[utl::MediaDescriptor::PROP_DOCUMENTTITLE] <<= aDocumentTitle;
                             rDescriptor[utl::MediaDescriptor::PROP_ASTEMPLATE] <<= !bIsLOK;
                             rDescriptor["RepairPackage"] <<= true;
+#if 0
                         }
                         else
                             rDescriptor["RepairAllowed"] <<= false; // Do not ask again
+#endif
                     }
                 }
             }
